include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -66,10 +66,10 @@ class SFML_AUDIO_API InputSoundFile : NonCopyable
     /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC, MP3.
     /// The supported sample sizes for FLAC and WAV are 8, 16, 24 and 32 bit.
     ///
-    /// Because of minimp3_ex limitation, for MP3 files with big (>16kb) APEv2 tag,
+    /// Because of dr_mp3 limitation, for MP3 files with big (>16kb) APEv2 tag,
     /// it may not be properly removed, tag data will be treated as MP3 data
     /// and there is a low chance of garbage decoded at the end of file.
-    /// See also: https://github.com/lieff/minimp3
+    /// See also: https://github.com/mackron/dr_libs
```

Comment:
We ought to double check this limitation or at least err on the side of caution and remove this disclaimer if we're not sure it's true.

Reply:
I agree. Actually wanted to make a draft PR since I had crashing MP3s, but then it somehow reverted to a normal PR again.

dr_mp3 however just wrapped minimp3's decoding functionality in a nicer API, so I'd assume thr same limitations apply.

Reply:
dr_mp3 still operates on 16kb buffers, which I assume is where this issue comes from. I tried to reproduce it by writing large APEv2 tags but was unable to get it to produce garbage.

Since ID3 tags are anyways more common than APEv2, at least in my experience, I'd say we don't necessarily have to document this edge case.